code/game/objects/items/his_grace.dm

```diff
@@ -47,6 +47,18 @@
 	else
 		..()
 
+/obj/item/his_grace/afterattack(atom/A, mob/user, proximity)
+	. = ..()
+	if(!proximity)
+		return
+	if(!QDELETED(A) && (istype(A, /obj/machinery/door) || istype(A, /obj/structure/door_assembly)))
```

Comment:
why not make all machinery and obj take more damage instead of just doors

Reply:
This is specifically geared towards making doors less effective, not windows or machines.